index.html

```diff
@@ -1590,19 +1575,23 @@ <h4>Time expressions</h4>
             <ul>
```

Comment:
Maybe we should say "A clock-time as restricted by this specification"?

Reply:
Equivalent done in 14589e1.